README.md

```diff
@@ -293,6 +309,30 @@ if (Meteor.isServer) {
 
 Other table settings should work normally, except that all fields will be sorted by value, even if using `fn`. The fields setting is required when using a server-side collection.
 
+### Settings
+
+Possible options:
+- autoQuoteSearchterm (Boolean - *default=* **true**):
+  This controlls if the searchterm should be automatically been qutoted for regex special chars. If you want to have pure RegEx possibility as searchterm, set this option to **false**.
```

Comment:
Well, you could use a regex as searchterm, like

me|you -> would give you records with me or you

or

[1-3]go -> would give you "1go", "2go", "3go" but not "5go", "6go" or "whatevergo"

It's regex with those mighty possibilities (but 99,9999% of worlds people would never use that). It might be something if you have additional search object (more input fields, filter options or something like that) and combine that in code, so the user does not have to speak "regex".